app/components/language-logo.ts

```diff
@@ -15,5 +15,6 @@ export default class LanguageLogoComponent extends Component<Signature> {}
 declare module '@glint/environment-ember-loose/registry' {
   export default interface Registry {
     LanguageLogo: typeof LanguageLogoComponent;
+    GleamLogo: typeof import('./gleam-logo').default;
```

Comment:
@Arpan-206 why is this registered here and not in `gleam-logo.ts` (like every other component is)?